### PR TITLE
Add a redoc API reference to the site

### PR DESCRIPTION
**Pull request by OisinKyne**

### CODE REVIEW

File: static/api-spec.yaml [hunk 1]
```diff
@@ -0,0 +1,754 @@
...
+              'tags': ['System'],
+            },
+        },
+      '/dv/{configHash}':
+        {
+          'get':
+            {
+              'operationId': 'DVController_getClusterDefinition',
+              'summary': 'Retrieve a Distributed Validator Cluster proposal',
+              'description': "This endpoint allows a charon client or launchpad interface to retrieve the terms of a proposed DKG. Once all operators listed in the DKG have submitted signed approvals to the terms, this object will be ready for a cluster of operators to use as part of a DKG ceremony. If the objects in the `operators` array are not fully populated, these operators need to use the [PUT request](#/Distributed-Validators/operation/DVController_updateClusterDefinition) to upload their charon client's public key and a signature from their address to indicate their acceptance of the terms.",
```
Comment (thomasheremans) [inline static/api-spec.yaml:207]:
```suggestion
              'description': "This endpoint allows a charon client or launchpad interface to retrieve the terms of a proposed DKG. Once all operators listed in the DKG have signed the terms of the cluster, this object will be ready for a cluster of operators to use as part of a DKG ceremony. If the objects in the `operators` array are not fully populated, these operators need to use the [PUT request](#/Distributed-Validators/operation/DVController_updateClusterDefinition) to upload their charon client's public key and a signature from their address to indicate their acceptance of the terms.",
```

Also, the PUT Request link does not work

File: static/api-spec.yaml [hunk 2]
```diff
@@ -0,0 +1,754 @@
...
+                        },
+                    },
+                },
+              'tags': ['Distributed Validators'],
+            },
+          'put':
+            {
+              'operationId': 'DVController_updateClusterDefinition',
+              'summary': 'Accept a proposed Distributed Validator Cluster',
+              'description': 'This endpoint is used by the operators present in the `operators` array of a cluster definition. These entites must submit a public key (in ENR form) to serve as their identity during the DKG, along with EIP712 signatures indicating their acceptance of the terms of this DKG.',
```
Comment (thomasheremans) [inline static/api-spec.yaml:241]:
```suggestion
              'description': 'This endpoint is used by the operators present in the `operators` array of a cluster definition. These operators must submit a public key (in ENR form) to serve as their identity during the DKG, along with EIP712 signatures indicating their acceptance of the terms of this DKG.',
```

Confusing to now refer to them as entities

**Review by thomasheremans — APPROVED** [review]
- A few suggestions in code
- Download button at the top does not work
- Should we redirect support to an email:support@obol.tech? I don't see this alias in our Google setup? Who does it refer to? Maybe discord is better as we mention in other places of the docs?
- For enquiries in photo below is empty; contact us details should be put there.
- I assume the version is completely independent of the rest of our versioning?

Screenshot 2023-02-08 at 21 11 42

**Review by corverroos — APPROVED** [review]

**Comment by OisinKyne** [comment]
> Download button at the top does not work

> Should we redirect support to an email:[support@obol.tech](mailto:support@obol.tech)? I don't see this alias in our Google setup? Who does it refer to? Maybe discord is better as we mention in other places of the docs?
    
Open API spec dictates this must be an email. https://swagger.io/specification/ Currently relying on the fact that any unmatched email comes my (and Angie's) way. Can setup an actual address that goes to people. Need to do the same thing with `security@`. 

> For enquiries in photo below is empty; contact us details should be put there.

Removed for now, we can put discord stuff there next. 

> I assume the version is completely independent of the rest of our versioning?

No this is the `config_version` we continue iterating on. So basically the internal versioning we care about between charon and the DVL. 

> Is it possible to remove the redocly banner in lower left? See second photo
I tried for a while and couldn't make it work. Eventually managed to get it mostly the right colour with some scrolling weirdness. This is free open source software I think we just rep it. Its much less branded than our existing [swagger UI](https://api.obol.tech/)

**Pull request merged**